itl/plugins-contrib.d/icingacli.conf

```diff
@@ -18,20 +30,34 @@ object CheckCommand "icingacli-businessprocess" {
 			set_if = "$icingacli_businessprocess_details$"
 			description = "Get details for root cause analysis"
 		}
-		"--state-type" = {
-			value = "$icingacli_businessprocess_statetype$"
-			description = "Define which state type to look at. Could be either soft or hard, overrides an eventually configured default"
-		}
+		"--downtime-is-ok" = {
+			set_if = "$icingacli_businessprocess_downtimeisok$"
+			description = "Treat hosts/services in downtime always as UP/OK."
+		}		
 		"--process" = {
 			value = "$icingacli_businessprocess_process$"
 			description = "Business process to monitor"
 			skip_key = true
 			required = true
 			order = -1
 		}
+		"--root-cause" = {
+			set_if = "$icingacli_businessprocess_rootcause$"
+			description = "Used in combination with --blame. Only shows the path of the nodes which are responsible for the state of the business process"
+		}
+		"--state-type" = {
+			value = "$icingacli_businessprocess_statetype$"
+			description = "Define which state type to look at. Could be either soft or hard, overrides an eventually configured default"
+		}
```

Comment:
Oh, and while you’re on it: please put this one back. (Yonas has already apologised for giving you an instruction which conflicted with mine.)

Reply:
moved back again